src/main/java/com/wolox/training/controllers/BookController.java

```diff
@@ -0,0 +1,18 @@
+package com.wolox.training.controllers;
+
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+public class BookController {
+
+    @GetMapping("/greeting")
+    public String greeting(@RequestParam(name="name", required=false, defaultValue="Stranger") String name, Model model) {
```

Comment:
Use `final` for all immutable parameters

Reply:
done. I don't fully understand how the `model` object is being modified once i declared as final tho